uint32 iterations

---

uint32 iterations
bool success
